apex/functions/sign-xpi/function.privileged.dev.json

```diff
@@ -0,0 +1,15 @@
+{
+  "runtime": "python",
+  "description": "The sign-xpi lambda with a set of credentials for privileged addons",
+  "timeout": 30,
+  "hooks": {
+    "build": "pip install -r requirements.txt -t site-packages"
+  },
+  "environment": {
+    "autograph_serverUrl": "http://52.91.167.168:8000/",
+    "autograph_hawkId": "alice",
+    "autograph_hawkSecret": "fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu",
```

Comment:
This particular set of credentials comes from the README for Autograph, and I think I can commit it as-is as a set of dev credentials. For stage/prod, I don't have a good answer for that question -- I'm leaving it up to ops. @jasonthomas, do you maybe have an opinion?